src/ducks/Studio/timeseries/fetcher.js

```diff
@@ -85,8 +85,17 @@ const transformAliases = [
   'percentOfTokenSupplyOnExchanges'
 ]
 
+SOCIAL_TWITTER_INTERVALS.forEach(interval => {
+  const queryKey = 'twitter_followers_' + interval
+
+  Fetcher[queryKey] = {
+    query: GET_METRIC({ key: 'twitter_followers', queryKey: queryKey })
+  }
+})
```

Comment:
It was stated before, that this code is not needed after using correct query. Query builder will take care of everything without additional interference

Reply:
I hope you sure, that key and queryKey will work correct for dinamic metrics :)

Reply:
I spent a lot of time for their combination and debug

Reply:
removed